cnxeasybake/tests/rulesets/deferred.less

```diff
@@ -0,0 +1,21 @@
+// Test before pseudo target
+
+div[data-type='chapter'] {
+    section.practice-test {
+        string-set: childclasses string(childclasses) " " attr(class);
+    }
+    section.key-equations {
+        string-set: childclasses string(childclasses) " " attr(class);
+    }
+
+    &:deferred {
```

Comment:
I see, you're thinking of body::before as not-an-ancestor or the page. But the context is actually 'body' which _is_ an ancestor. The next problem, though is that the current hack doesn't allow it, since I didn't extend tinycss2, which chokes on unknown pseudo classes, instead I catch the error and turn it into a pseudo-element, which pass through  just fine, soooo, I guess I better file a bug to go do it correctly.